cap/modules/schemas/views.py

```diff
@@ -118,60 +120,71 @@ def permissions(name=None, version=None):
 
 
 @blueprint.route(
-    '/<string:name>/notifications', methods=['GET', 'PATCH']
+    '/<string:name>/notifications',
+    methods=['GET', 'PATCH', 'DELETE', 'POST', 'PUT']
 )
 @blueprint.route(
     '/<string:name>/<string:version>/notifications',
-    methods=['GET', 'PATCH'],
+    methods=['GET', 'PATCH', 'DELETE', 'POST', 'PUT'],
 )
 @get_schema
 @schema_admin_permission
 @super_admin_permission.require(http_exception=403)
 def notifications_config(name=None, version=None, schema=None, *args, **kwargs):
-    """CRUD operations for schema configuration."""
+    """CRUD operations for schema notification configuration."""
```

Comment:
lets create a generic method that can take care of `schema.config` updates (so not explicit to notifications). this way it will be easier to implement new endpoints for other config types and without duplicates